### ISSUE TITLE
Command [count]o can't repeat in vimwiki.

### ISSUE THREAD

**Issue by ChenWeird** [vimwiki/vimwiki#776]
**1. Checkout the `dev` branch and confirm the issue is present there as well.
   The `dev` branch contains fixes that may not have been merged to `master` yet.**

```Plug 'vimwiki/vimwiki', { 'branch': 'dev' }```

**2. Post the syntax you are using (default/mediawiki/markdown) **and** your vimwiki settings from your `.vimrc`**

syntax: default
vimrc:
```
let wiki = {}
let wiki.name = 'Personal'
let wiki.path = 'd:/Vimwiki/'
let wiki.ext = '.wiki'
let wiki.syntax = 'default'
let wiki.nested_syntaxes = {'python': 'python'}
let wiki.links_space_char = '_'
" If use Markdown or MediaWiki syntax, set this option to 0, else to 1
let wiki.list_margin = 1
let wiki.auto_toc = 1
let wiki.auto_tags = 1
let wiki.auto_generate_tags = 1

let g:vimwiki_list = [wiki]
let g:vimwiki_auto_chdir = 1
```
**3. Provide a detailed description of the problem including **steps to reproduce the issue**.**
```
1.  <leader>ww
2.  5o<esc>
```
_**There's only one new line,  didn't repeat [count] times.**_
But it can work outside vimwiki.

**4. Include the output of `:VimwikiShowVersion`.**

```
Branch: dev
Revision: 2411266
Date: 2019-10-28 05:05:03 -0600
```

**Comment by ranebrown** [comment]
This occurs because vimwiki remaps `o` and `O` to properly insert list markers when within a list. I think it should be easy to fix.

**Comment by tinmarino** [comment]
Fixed: f49a9150
Just repeating the command `call vimwiki#lst#kbd_o()`, `v:count` times solves consistently the problem.
I created a helper function some other map could benefit. 
```vim
" Execute: string v:count times
function! vimwiki#u#count_exe(cmd) abort
    for i in range( max([1, v:count]) )
        exe a:cmd
    endfor
endfunction
```
